test/unit/lifecycle/properties.js

```diff
@@ -25,7 +26,11 @@ describe('lifecycle/property', () => {
   });
 
   describe('props declared as attributes with ES2015 classes are linked', () => {
+    const skip = !classStaticsInheritance();
```

Comment:
I'm saying `Object.setPrototypeOf` is ES6, class is also ES6. So babel compiles class to `Object.setPrototypeOf`, not some ES5 code?